sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/AnalysisSuite.scala

```diff
@@ -1051,69 +1051,69 @@ class AnalysisSuite extends AnalysisTest with Matchers {
   }
 
   test("SPARK-22748: Analyze __grouping__id as a literal function") {
-    assertAnalysisSuccess(parsePlan(
+    assertAnalysisSuccess(EliminateSubqueryAliases(parsePlan(
```

Comment:
why do we need to remove subquey alias?

Reply:
I just found that the query:
```sql
SELECT * FROM (
  SELECT a, b, count(1) FROM TaBlE2
    GROUP BY a, b GROUPING SETS ((a, b), ())
) WHERE grouping__id > 0
```

was actually wrong, as it missed selecting `grouping__id` in the subquery.